functions/src/portal-types.ts

```diff
@@ -0,0 +1,82 @@
+export type UserType = "student" | "teacher";
```

Comment:
I tried importing these definitions from the original of this file but it confused the build process for the functions, so I ended up making a duplicate of the file here.

Reply:
I think @dougmartin and/or @pjanik ran into something like this before and found a fix.